ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -228,6 +255,19 @@ mac_address_table:
             - < hostname1 >
             - < hostname2 >
 
+        # Explicitly enable or disable evpn_l3_multicast to override setting of tenants.<tenant>.evpn_l3_multicast.enabled.
+        # Allow override of tenants.<tenant>.evpn_l3_multicast.node_settings
+        evpn_l3_multicast:
+          enabled: < true | false >
+          evpn_peg:
+            # For each group of nodes, allow configuration of EVPN PEG features | Optional
+            - nodes: [ < node_1 >, < node_2 >, < node_N > ]                # Optional - will apply to all nodes with RP addresses configured if not set.
+              transit: < true | false | default false >                    # Enable EVPN PEG transit mode
+        rp_addresses:                                                    # For each group of nodes, allow configuration of RP Addresses & associated groups
```

Comment:
Should be updated similar to tenant config. `pim_rp_addresses`

Reply:
Thanks, my documentation update was really sloppy - my bad!